modules/swagger-codegen/src/main/resources/rust/configuration.mustache

```diff
@@ -1,10 +1,23 @@
 {{>partial_header}}
 use hyper;
+use std::collections::HashMap;
```

Comment:
Is this not going to yield warnings? It might be nice to allow the warning or only add this `use` statement if necessary.

Reply:
It would, but at this point we're too far gone on warnings.

I would like to do a pass dedicated to squashing warnings at some point, which I think will be easiest with a certain amount of refactoring. For now, I'm okay with living with another warning in the sea of them.

Reply:
To be specific, I think refactoring the shared code (the whole `fn operationId` bit) to be one generic function which takes configurable things as parameters would result in more readable code that would be easier to eliminate warnings from... if only because all the code would be there and just not be called necessarily. Each operation would then just be a short function calling the big generic "do_call" style function with appropriate args.

I recall some of the other clients take this approach, but I don't have references handy because I dug into this some months ago.